assignments/prototypes.js

```diff
@@ -102,7 +140,7 @@
   console.log(archer.greet()); // Lilith offers a greeting in Elvish.
   console.log(mage.takeDamage()); // Bruce took damage.
   console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
```

Comment:
the work meets mvp, but I would like to see you going beyond MVP. I know you had family issues so I understand not being able to hit some stretches, but I'm definitely come to expect stretches from your work. Stay strong. 💪 ⁉️ 🌟